src/components/paneler/innhold/personaliaV2/kontonummer.tsx

```diff
@@ -14,9 +15,15 @@ function Kontonummer(props: { kontonummer?: string }) {
 
 	if (props.kontonummer) {
 		content = (
-			<Normaltekst className="innrykk">
+			<Normaltekst className="innrykk display-flex">
 				{kontonummer}
-				<Kopiknapp kopitekst={props.kontonummer} type="kontonummer" />
+				<CopyToClipboard
+					copyText={props.kontonummer}
+					popoverText="Kopiert"
```

Comment:
Samme begrunnelse som i forrige kommentar:

```suggestion
					popoverText="Kontonummer ble kopiert"
```